bin/ch/ChakraRtInterface.cpp

```diff
@@ -4,7 +4,12 @@
 //-------------------------------------------------------------------------------------------------------
 #include "stdafx.h"
 
-LPCWSTR chakraDllName = _u("chakracore.dll");
+#ifdef _WIN32
+LPCSTR chakraDllName = "chakracore.dll";
+#else
+#include <dlfcn.h>
+LPCSTR chakraDllName = "libChakraCoreLib.so";
```

Comment:
Done- I had originally named the target as ChakraCoreLib to avoid conflicting with the project name, but switched them to be the same and it seems to work :)